tutorials/data_management/cloud_storage/binding_with_azure/chapter.md

```diff
@@ -13,21 +13,29 @@ We assume you already have an Azure account with resource group and storage acco
 1. Create a Container in the created Storage account  
```

Comment:
There is no section at the end regarding how can i see that the subscription was created properly
Go back home-> go to Event Subscriptions ->. enter the correct region for you function , subscription and topic type 
make sure you see your newly created subscription